src/components/results/detail/VideoDetail.vue

```diff
@@ -32,11 +32,12 @@
                   <v-col>
                     <div class="text-body-1">
                       <video-player
-                        :options="$data.videoOptions"
+                        :options="videoOptions"
```

Comment:
It is an opinion, but I think it is better to generally use @data and @props in the template, so that it gets easier to locate the origin of properties. With many mixins, props, data, computed, methods it can get quite messy.

Reply:
oh i see, nvm, you turned it into a computed.